fsspec/implementations/tests/test_zip.py

```diff
@@ -30,6 +30,8 @@ def test_empty():
     with tempzip() as z:
         fs = fsspec.get_filesystem_class("zip")(fo=z)
         assert fs.find("") == []
+        with pytest.raises(FileNotFoundError):
+            fs.info("")
```

Comment:
One could argue that the root always exists and is a directory-like thing, since you can list it.

Reply:
Hm, I'd say
- yes, ls("") should of course work
- therefore, info("") should, I suppose, return a directory - but after some thought, I am less certain. One would expect isdir("") == True, for example, but we don't have any permissions or other info for this entry as we do for any other dir in the archive
- find("") should *not* include "" itself, even for withdirs=True, you only get children

Reply:
@martindurant, I left the behavior as it was before. I think if we need that to change we can open a separate issue/PR.